src/openforms/js/components/admin/form_design/registrations/zgw/OptionsFormFields.js

```diff
@@ -0,0 +1,331 @@
+import {produce} from 'immer';
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {useIntl} from 'react-intl';
+
+import {CustomFieldTemplate} from 'components/admin/RJSFWrapper';
+import {TextArea, TextInput} from 'components/admin/forms/Inputs';
+import Select from 'components/admin/forms/Select';
+import {ValidationErrorContext} from 'components/admin/forms/ValidationErrors';
+
+import {VariablePropertyModal} from './OptionsVariablesProperties';
+import {getChoicesFromSchema, getErrorMarkup, getFieldErrors} from './utils';
+
+const Wrapper = ({children}) => (
+  <form className="rjsf" name="form.registrationBackendOptions">
+    <CustomFieldTemplate displayLabel={false} errors={null}>
+      <fieldset id="root">{children}</fieldset>
+    </CustomFieldTemplate>
+  </form>
+);
+
+const ZGWFormFields = ({index, name, schema, formData, onChange}) => {
+  const intl = useIntl();
+  const validationErrors = useContext(ValidationErrorContext);
+
+  const {
+    zgwApiGroup = '',
+    zaakVertrouwelijkheidaanduiding = '',
+    zaaktype = '',
+    informatieobjecttype = '',
+    organisatieRsin = '',
+    medewerkerRoltype = '',
+    objecttype = '',
+    objecttypeVersion = '',
+    contentJson = '',
+  } = formData;
+
+  const onFieldChange = event => {
+    const {name, value} = event.target;
+    const updatedFormData = produce(formData, draft => {
+      draft[name] = value;
+    });
+    onChange(updatedFormData);
+  };
+
+  return (
+    <Wrapper>
+      <CustomFieldTemplate
+        id="root_zgwApiGroup"
```

Comment:
What is the reason for these ids (instead of for example `registrationBackendOptions.zgwApiGroup`)?

Reply:
This id here is used in `CustomFieldTemplate` component for the `htmlFor` property of the label. Otherwise it's not accessible. If I use `registrationBackendOptions.zgwApiGroup` I have to change it in the `Select` component below, correct?